http/api/transport.go

```diff
@@ -176,7 +180,7 @@ func encodeError(_ context.Context, err error, w http.ResponseWriter) {
 	}
 
 	if errorVal, ok := err.(errors.Error); ok {
-		w.Header().Set("Content-Type", contentType)
+		w.Header().Set("Content-Type", contentTypeJSON)
```

Comment:
Why only JSON, what about other 3 CTs we support?

Reply:
Because it's the CT of the error response